scripts/bash/help_script.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
```

Comment:
One more topic if we are doing several improvements in one PR (again :-| ).
What about the file name(s)?
I think `help_script.sh` is totally out of the pattern of others.
But furthermore, probably would make it easier for orientation to prefix all partial scripts with `_` . And keep only `run_standardization.sh`, `run_conformance.sh` and `run_standardization_conformance.sh` as they are now (and perhaps the `enceladus_env.sh`, not to introduce a breaking change).

Reply:
> One more topic if we are doing several improvements in one PR (again :-| ).
Is it? It is detached mode + help (as documentation)

I did `_print_help.sh`. I am thinking of also pulling all function into `_functions.sh`

Reply:
> 
> 
> > One more topic if we are doing several improvements in one PR (again :-| ).
> 
> Is it? It is detached mode + help (as documentation)

+ Refactoring + asynchronous. But hey I don't mind. It's cool it's getting better 😉 🥇